### PR TITLE
refactor: clean up login endpoint

### PR DESCRIPTION
**Pull request by hypersolo**
## What does this MR do?

we follow snake_case in req and res objects.
it wasn't followed in login endpoint. i fixed it in this pr <3

## Check list  

- [ ] All Merge Request pipelines should be passed.
- [ ] Make sure all the changes are necessary and correct.
- [ ] Make sure no needed change missing (all necessary changes are made).
- [ ] Merge Request should contains only changes for one purpose (one feature or bug or refactoring).
- [ ] All necessary unit tests are written.
- [ ] Critical changes should be tested locally (or on staging).

### CODE REVIEW

File: src/presentation/http/controllers/auth.controller.ts [hunk 1]
```diff
@@ -21,42 +21,43 @@ export class LoginController extends BaseController {
...
     this.authService.validateSignature({
-      userAddress: user_address,
+      userAddress,
       signature,
     });
-    let userOrders, userFills;
+    let orders, fills;
     if (user_data === true) {
-      userOrders = await this.orderRepo.getByUser({
+      orders = await this.orderRepo.getByUser({
```
Comment (WilliamHawkingStephen) [inline src/presentation/http/controllers/auth.controller.ts]: - its might be better to use promise for getting order and fill in parallel : 
`
 const [orders, fills] = await Promise.all([
      this.orderRepo.getByUser({ network, userAddress }),
      this.fillRepo.getByUser({ network, userAddress }),
    ]);
`

**Comment by WilliamHawkingStephen** [comment]
- great job broski :heart:

File: src/presentation/http/controllers/auth.controller.ts (hunk 1 above)
Comment (hypersolo) [reply src/presentation/http/controllers/auth.controller.ts]: yup you are right, https://github.com/dexpresso-protocol/core/pull/576/commits/fc81b13e26fb127de9ae7f067bb8fa24702f07f5

**Review by WilliamHawkingStephen — APPROVED** [review]

**Review by franken5tein — APPROVED** [review]

**Review by GreatGaryCooper — APPROVED** [review]
LGTM

**Pull request merged**